src/beautifiers/r-beautifier/r-beautifier.r

```diff
@@ -0,0 +1,4 @@
+if(!("formatR" %in% installed.packages()[,"Package"])) install.packages("formatR",repos='http://cran.us.r-project.org')
+library(formatR)
+file <- commandArgs(trailingOnly = TRUE)[1]
+tidy_source(text = readLines(file))
```

Comment:
It would be nice to add support for some of the available options: http://yihui.name/formatR/

![image](https://cloud.githubusercontent.com/assets/1885333/17569579/d6846c76-5f16-11e6-84c0-0575b6e363b1.png)

The `indent` option being the most important.